src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -309,11 +313,44 @@ function NodeComponent({
 
                         {params.nodeType === NodeType.Pod.toLowerCase() && (
                             <div className={'flex left col-1 pt-9 pb-9'}>
-                                {' '}
-                                {node.info?.filter((_info) => _info.name === 'Containers')[0]?.value}{' '}
+                                {node.info?.filter((_info) => _info.name === 'Containers')[0]?.value}
+                            </div>
+                        )}
+                    
+                        {params.nodeType === NodeType.Pod.toLowerCase() && (
+                            <div className={'flex left col-1 pt-9 pb-9'}>
+                                {(node.kind !== 'Containers') && (node.info?.map((_info) => _info.name === 'Restart Count' ?_info.value : 0).sort((a,b) => a > b ? -1: 1)[0])}
```

Comment:
If info can be optional then put optional before sort as well.

Reply:
Also, we already have some helpers written for sorting so can you please check and try to use them if possible?